src/internals/ir/callGraph.ts

```diff
@@ -263,4 +280,31 @@ export class CallGraph {
     }
     return undefined; // e.g. self.<map_field>.set()
   }
+
+  /**
+   * Checks if a function is an extension function with the "extends" attribute.
+   * @param func The function definition to check.
+   * @returns True if the function has the "extends" attribute.
+   */
+  public static isExtensionFunction(func: AstFunctionDef): boolean {
```

Comment:
I would move this method and the method below to Tact utils. It seems to be generally useful and unrelated to the callgraph logic.